src/Sulu/Component/Rest/RequestParametersTrait.php

```diff
@@ -49,7 +53,7 @@ protected function getRequestParameter(Request $request, $name, $force = false,
      * @param bool $force TRUE if value is mandatory
      * @param bool $default value if parameter not exists
      *
-     * @return bool
+     * @return bool|null
```

Comment:
This could be improved to let phpstan if default is bool it always return a bool:

```
	/**
     * @template T of bool|null
     *
	 * @param T $default
     *
	 * @return bool|T
	 */
```

https://phpstan.org/r/01d59854-daa7-4cb7-b003-8fe8ac17101c

Reply:
Yeah that looks better.